Tutorials/Make_Your_Own_Intake_Datastore.ipynb

```diff
@@ -0,0 +1,470 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        columns_with_iterables=["variable"] # This is important
</pre><p>This should be changed to <code>columns_with_iterables=list(builder.columns_with_iterables)</code> since there are actually multiple columns with iterables.</p><p> </p><p>Or, perhaps more generally, it should be changed to</p>

<pre class="ql-syntax" spellcheck="false">        columns_with_iterables=[
            "variable",
            "variable_long_name",
            "variable_standard_name",
            "variable_cell_methods",
            "variable_units",
] # This is important
</pre><p>so that it can be copied elsewhere</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/440/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
ok done